### ISSUE TITLE
Update to angular 8

### ISSUE THREAD

**Issue by jpike88** [valor-software/ng2-charts#1114]
```ng2-charts@2.2.3 requires a peer of @angular/common@^7.2.0 but none is installed. You must install peer dependencies yourself```

**Comment by paviad** [comment]
Why is this necessary? I just created a minimal angular app with angular 8 and it works out of the box, here's the package.json

```
{
  "name": "ch1",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.0.0",
    "@angular/common": "~8.0.0",
    "@angular/compiler": "~8.0.0",
    "@angular/core": "~8.0.0",
    "@angular/forms": "~8.0.0",
    "@angular/platform-browser": "~8.0.0",
    "@angular/platform-browser-dynamic": "~8.0.0",
    "@angular/router": "~8.0.0",
    "chart.js": "^2.8.0",
    "ng2-charts": "^2.2.3",
    "rxjs": "~6.4.0",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.800.0",
    "@angular/cli": "~8.0.0",
    "@angular/compiler-cli": "~8.0.0",
    "@angular/language-service": "~8.0.0",
    "@types/node": "~8.9.4",
    "ng2-charts-schematics": "^0.1.7",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  }
}
```

**Comment by paviad** [comment]
Anyway, updated the peer dependencies for correctness' sake.

Closed by b7c2226
